src/main/resources/database/student.sql

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Why do we need sql queries to update manually from application. Springboot application will create tables automatically into dabase with entities defined in application.